specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1453 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkVerifierManagementClient",
+    "version": "2022-06-01-preview"
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkVerifiers/{networkVerifierName}/analysisIntents": {
+      "get": {
+        "tags": [
+          "AnalysisIntent"
+        ],
+        "summary": "Gets the list of Analysis Intent resources",
+        "operationId": "AnalysisIntent_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
```

Comment:
>./network.json#/parameters/ApiVersionParameter [](http://example.com/codeflow?start=21&length=46)

Use ApiVersionParameter definition from v3 common types for this. This comment applies to the whole PR.